### PR TITLE
[FSSDK-10839] prepare for release 4.2.0

### PR DESCRIPTION
**Pull request by raju-opti**
## Issues
- FSSDK-10839

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Maven version
-version = 3.1.0-SNAPSHOT
```
Comment (jaeopt) [inline gradle.properties:2]: We not need to manage this version.

Comment (jaeopt) [reply gradle.properties:2]: Can we confirm all java-sdk changes compatible with android-sdk?

**Review by jaeopt — COMMENTED** [review]
Looks good. I'd like to confirm all the changes work with android-sdk.

File: gradle.properties (hunk 1 above)
Comment (raju-opti) [reply gradle.properties]: Updated and tested with android sdk

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**